src/custom/components/AffiliateStatusCheck/index.tsx

```diff
@@ -71,18 +65,6 @@ export default function AffiliateStatusCheck() {
       return
     }
 
-    try {
```

Comment:
I'm not sure if removing this is the best approach since if not the state would be active and that's not what we want./